app/controllers/notifications_controller.rb

```diff
@@ -239,7 +239,7 @@ def selected_notifications
     if params[:id] == ['all']
       current_notifications
     else
-      current_user.notifications.where(id: params[:id])
+      current_user.notifications.unmuted.where(id: params[:id])
```

Comment:
Same as above, let's keep the default scope the same as before.

Reply:
will update